src/dao/user.js

```diff
@@ -25,10 +25,10 @@ const createUserDAO = async (username, password, email, salt) => {
 
 const checkExistingUserOrEmail = async (username, email) => {
   const result = await db.raw(`select count (*) from users where username = '${username}' or email = '${email}' `)
-  if (result.rows[0].count == 0) {
-    return false
-  } else {
+  if (result.rows[0].count > 0) {
```

Comment:
this one you dont read the comment, you can simplyfy it
just like

return result.rows[0].count > 0